mysql/const.go

```diff
@@ -285,7 +285,9 @@ var DefaultLengthOfMysqlTypes = map[byte]int{
 	TypeFloat:    4,
 	TypeDouble:   8,
 
+	TypeEnum:   1,
```

Comment:
Is this the same with MySQL? See http://dev.mysql.com/doc/refman/5.7/en/storage-requirements.html